differ/storage.go

```diff
@@ -557,9 +598,9 @@ func (storage DBStorage) CleanupForOrganization(orgID types.OrgID, maxAge string
 
 	log.Info().
 		Str(MaxAgeAttribute, maxAge).
-		Str("delete statement", printableStatement).
+		Str("DeleteStatement", printableStatement).
```

Comment:
DeleteStatement without the double quota

Reply:
ah sure